src/httpClient.test.js

```diff
@@ -175,48 +175,77 @@ describe(`httpClient`, function() {
     })
   }.bind(this))
 
+    it(`executes the beforeRun function passing itself as the argument`, () => {
```

Comment:
I find this superceded bu the following one. I think it's best to remove it. Please delete it, bump version and merge.